config/config.go

```diff
@@ -1004,7 +1003,7 @@ func NewConfig() (c *Config) {
 					Otel: OtelCollector{
 						CAName:     "",
 						Protocol:   "http",
-						SkipVerify: true,
+						SkipVerify: false,
```

Comment:
I agree with this change. It should match the documented default, and I think we do want `true`, `false` is less secure.